03.bowling/bowling.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  shots << if s == 'X'
+             10
+           else
+             s.to_i
+           end
+end
+
+frame = 0
+max_frame = 10
+count = 0
+result = [0] * max_frame
+
+shots.length.times do |i|
```

Comment:
```suggestion
shots.each_with_index do |shot, i|
```

として、`shots[i]`は`shot`に置き換えると可読性が上がりそうです